pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1959,6 +1958,7 @@ func parseConfigItems(config *zconfig.EdgeDevConfig, ctx *getconfigContext) {
 
 		// Set GlobalStatus Values from GlobalConfig.
 		newGlobalStatus.UpdateItemValuesFromGlobalConfig(*gcPtr)
+		ctx.zedagentCtx.globalStatus = newGlobalStatus
```

Comment:
Great catch.. Thought this was specific to ConfigItemChanges I am testing out.. Did not realize it was already there before it, from the time we introduced the ConfigItemStatus..